bigquery_etl/cli/utils.py

```diff
@@ -104,6 +104,14 @@ def paths_matching_name_pattern(pattern, sql_path, project_id, files=("*.sql")):
     callback=is_valid_dir,
 )
 
+tmp_dir_option = click.option(
+    "--tmp_dir",
+    "--tmp-dir",
+    help="Path to temporary directory generated SQL is written to.",
+    type=click.Path(file_okay=False),
+    default=Path("/tmp/sql"),
```

Comment:
Can we consider generating the tmp directory name rather than a static path?

Reply:
A temporary directory is now generated unless explicitly specified